### PR TITLE
git.io->cloudposse.tools update

### PR DESCRIPTION
**Pull request by dylanbannon**
## what and why
Change all references to `git.io/build-harness` into `cloudposse.tools/build-harness`, since `git.io` redirects will stop working on April 29th, 2022.

## References
- DEV-143

### CODE REVIEW

File: .github/workflows/auto-readme.yml [hunk 1]
```diff
@@ -15,21 +17,35 @@ on:
...
 
+    - name: Find default branch name
+      id: defaultBranch
+      shell: bash
+      env:
+        GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+      run: |
+        default_branch=$(gh repo view --json defaultBranchRef --jq .defaultBranchRef.name)
+        printf "::set-output name=defaultBranch::%s\n" "${default_branch}"
+        printf "defaultBranchRef.name=%s\n" "${default_branch}"
```
Comment (mcalhoun) [inline .github/workflows/auto-readme.yml:33]: I think you can get this with `${{ github.event.repository.default_branch }}`

**Review by mcalhoun — APPROVED** [review]
LGTM, just left one small comment

File: .github/workflows/auto-readme.yml (hunk 1 above)
Comment (dylanbannon) [reply .github/workflows/auto-readme.yml:33]: Haha, awesome. I didn't expect this to show up on this PR, but thanks for the feedback and I'll improve it.

Comment (Nuru) [reply .github/workflows/auto-readme.yml:33]: @mcalhoun @dylanbannon `github.event.repository.default_branch` is not available when, as in this workflow, the `event` is `schedule`.

**Pull request merged**